README.md

```diff
@@ -10,8 +10,42 @@ Sails.js/Waterline adapter for MongoDB.
 > As an adapter, this module implements a set of declarative interfaces, conventions, and best-practices for integrating with Mongo databases.
 > Strict adherence to an adapter specification enables the (re)use of built-in generic test suites, standardized documentation, reasonable expectations around the API for your users, and overall, a more pleasant development experience for everyone.
 
+# BREAKING CHANGES
 
-## Installation
+## MongoDB Driver
+From `sails-mongo` version 2.0.0 and above, the adapter uses a new mongodb driver. These driver changes the way it handles connections,
+and implements the concept of [MongoClient](http://mongodb.github.io/node-mongodb-native/3.2/api/MongoClient.html).
+
+Because of that, `manager` now returns MongoClient, instead of just a connection.
```

Comment:
This worries me a bit -- I think it would affect the implementation of .native() and a number of places in our docs, in the reference section (and I think in one of the tutorials)

I know we talked about it but I forgot what we came up with (and now I can't find the conversation).  What do you reckon?

Reply:
@luislobo once we chat about this a bit more and sure we're on the same page, I'll re-review just to lend another pair of eyes to the actual meat of things below

Reply:
OK for posterity:  Luis totally talked about this with me in detail, I just have noodles for brains.

![image](https://user-images.githubusercontent.com/618009/57328359-d592f100-70d6-11e9-9968-4760e795383f.png)
![image](https://user-images.githubusercontent.com/618009/57328375-e04d8600-70d6-11e9-9e28-a039cb6d9ef7.png)
![image](https://user-images.githubusercontent.com/618009/57328411-fb1ffa80-70d6-11e9-9bc2-7434f6e80935.png)